### ISSUE TITLE
EAS Build Failing during prebuild: no such file NotificationService.h

### ISSUE THREAD

**Issue by tonymckendry** [OneSignal/onesignal-expo-plugin#66]
I am attempting to build a custom dev client for Expo with EAS Build and my iOS build is failing during the "Prebuild" stage with the following error: 
```
[stderr] onesignal-expo-plugin: Couldn't read file:node_modules/onesignal-expo-plugin/build/support/serviceExtensionFiles/NotificationService.h
- Config syncing
[stderr] Error: ENOENT: no such file or directory, open 'node_modules/onesignal-expo-plugin/build/support/serviceExtensionFiles/NotificationService.h'
```

onesignal-expo-plugin: 1.0.0-beta11
react-native-onesignal: 4.3.7

**Comment by rgomezp** [comment]
Howdy,
Perhaps try running `expo prebuild --clean`

**Comment by phi-tz** [comment]
This issue is happening for me as well.  Most likely because i'm in a monorepo.

**Comment by tonymckendry** [comment]
@phi-tz I was in a monorepo as well - I could never figure it out so I transitioned to using the Expo/EAS Push Notification service instead and it was quick and simple. The painful part was that we were migrating from a React Native app that already had onesignal implemented and we had to redo everything because OneSignal wouldnt work and I couldn't get anyone to respond and help me out. Oh well I guess, their loss of a customer.
